pages/linux/pro.md

```diff
@@ -0,0 +1,28 @@
+# pro
+
+> Manage Ubuntu Pro services.
+> More information: <https://manpages.ubuntu.com/manpages/latest/en/man1/ubuntu-advantage.1.html>.
+
+- Connect your system to the Ubuntu Pro support contract:
+
+`sudo pro attach`
+
+- Display the status of Ubuntu Pro services:
+
+`pro status`
+
+- Check if the system is affected by a specific vulnerability (and apply a fix if possible):
+
+`pro fix {{CVE-number}}`
+
+- Display the number of unsupported packages:
+
+`pro security-status`
+
+- List packages that are no longer available for download:
+
+`pro security-status --unavailable`
+
+- List third-party packages:
+
+`ubuntu-security-status --thirdparty`
```

Comment:
```suggestion
`pro ubuntu-security-status --thirdparty`
```
Shouldn't `pro` be present here too?

Reply:
Oh right, this is the correct command (it replaces `ubuntu-security-status`):
```suggestion
`pro security-status --thirdparty`
```